---
title: "Downstream Analysis - Significant Genes"
author: "Payal Banerjee"
date: "12/14/2020"
output: html_document
---

# What to do with significant differentially expressed genelist? 

## Functional Enrichment Analysis

[//]: # Create Markdown tables using this link -  https://www.tablesgenerator.com/markdown_tables
[//]: # Always keep a line in between table header and table other wise markdown doesn't render properly

| Name       	| Description                                                                                                                                                                                                                                                     	| Link                                       	|
|------------	|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|--------------------------------------------	|
| g:Profiler 	| Performs functional enrichment analysis, also known as over-representation analysis (ORA) or gene set enrichment analysis,<br>on input gene list. It maps genes to known functional information sources and detects statistically significantly enriched terms. 	| https://biit.cs.ut.ee/gprofiler/gost       	|
| DAVID      	| Provides a comprehensive set of functional annotation tools for investigators to understand biological meaning behind large list of <br>genes.                                                                                                                  	| https://david.ncifcrf.gov/                 	|
| Enrichr    	| A suite of gene list enrichment analysis tools                                                                                                                                                                                                                  	| https://maayanlab.cloud/Enrichr/           	|
| REViGO     	| Takes long lists of Gene Ontology terms and summarize them by removing redundant GO terms. The remaining terms can be visualized in semantic similarity-based scatterplots,interactive graphs, or tag clouds.                                                   	| http://revigo.irb.hr/                      	|
| WebGestalt 	| Functional enrichment analysis web tool                                                                                                                                                                                                                         	| http://www.webgestalt.org/                 	|
| GSEA       	| A computational method that determines whether a priori defined set of genes show significant, concordant differences between two biological states                                                                                                             	| https://www.gsea-msigdb.org/gsea/index.jsp 	|
| Harmonizome       	| Search for genes or proteins and their functional terms extracted and organized from over a hundred publicly available resources                                                                                                             	| https://maayanlab.cloud/Harmonizome/ 	|

## Pathways

| Name          	| Description                                                                                                                                                                                                                                                                                                                                   	| Link                                                	|
|---------------	|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|-----------------------------------------------------	|
| KEGG          	| KEGG is a database resource for understanding high-level functions and <br>utilities of the biological system, such as the cell, the organism and <br>the ecosystem, from molecular-level information, especially large-scale <br>molecular datasets generated by genome sequencing and other <br>high-throughput experimental technologies.  	| https://www.genome.jp/kegg/                         	|
| Reactome      	| REACTOME is an open-source, open access, manually curated and <br>peer-reviewed pathway database. Our goal is to provide intuitive <br>bioinformatics tools for the visualization, interpretation and analysis <br>of pathway knowledge to support basic and clinical research, genome <br>analysis, modeling, systems biology and education. 	| https://reactome.org/                               	|
| Wiki Pathways 	| WikiPathways is a database of biological pathways maintained by and for the scientific community.                                                                                                                                                                                                                                             	| https://www.wikipathways.org/index.php/WikiPathways 	|


## Interaction Databases

| Name      	| Description                                                                                    	| Link                    	|
|-----------	|------------------------------------------------------------------------------------------------	|-------------------------	|
| BioGRID   	| A biomedical interaction repository with data compiled through comprehensive curation efforts. 	| https://thebiogrid.org/ 	|
| GeneMANIA 	| Helps predict the function of your favourite genes and gene sets.                              	| https://genemania.org/  	|
| STRING    	| Protein-Protein Interaction Networks                                                           	| https://string-db.org/  	|


## Text Mining

| Name     	| Description                                                                                                                                                                                                                                                        	| Link                                  	|
|----------	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|---------------------------------------	|
| Geneshot 	| Geneshot finds publications that mention both the search terms and genes.It then prioritizes these genes using various methods: 1) list of genes from publications;<br>2) predicted genes using gene-gene similarity matrices derived from a variety of resources. 	| https://amp.pharm.mssm.edu/geneshot/  	|
| GLAD4U   	| GLAD4U allows you to query the scientific literature to retrieve and prioritize a list of Entrez-Gene IDs. Its ranking algorithm relies on <br>the hypergeometric test.                                                                                            	| http://glad4u.zhang-lab.org/index.php 	|
| PubMed   	| PubMedÂ® comprises more than 30 million citations for biomedical literature from MEDLINE, life science journals, and online books.                                                                                                                                  	| https://pubmed.ncbi.nlm.nih.gov/      	|


## Drug Interaction Database

| Name  	| Description                                                                                                     	| Link                   	|
|-------	|-----------------------------------------------------------------------------------------------------------------	|------------------------	|
| DGIdb 	| An open-source project.Drug-gene interactions and the druggable genome, mined from over thirty trusted sources. 	| https://www.dgidb.org/ 	|

## RNA seq specific Databases

| Name                    	| Description                                                                                                                                                                                                                         	| Link                            	|
|-------------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|---------------------------------	|
| GTEx                    	| A comprehensive public resource to study tissue-specific gene expression and<br>              regulation.                                                                                                                           	| https://gtexportal.org/home/    	|
| ARCHS4                  	| Massive Mining of Publicly Available RNA-seq Data from Human and Mouse                                                                                                                                                              	| https://maayanlab.cloud/archs4/ 	|
| Hipposeq                	| Interactively analyze and visualize RNA-seq data for all excitatory cell<br> populations in the hippocampus at multiple levels of granularity.                                                                                      	| https://hipposeq.janelia.org/   	|
| The Human Protein Atlas 	| Maps all the human proteins in cells, tissues and organs using an <br>integration of various omics technologies, including antibody-based <br>imaging, mass spectrometry-based proteomics, transcriptomics and systems<br> biology. 	| https://www.proteinatlas.org    	|

## Data Share

| Name 	| Description                                                                                                                                                                                                                                                                                                                                                                                                                    	| Link                              	|
|------	|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|-----------------------------------	|
| GEO  	| GEO is a public functional genomics data repository supporting <br>MIAME-compliant data submissions. Array- and sequence-based data are <br>accepted.                                                                                                                                                                                                                                                                          	| https://www.ncbi.nlm.nih.gov/geo/ 	|
| SRA  	| Sequence Read Archive (SRA) data, available through multiple cloud providers and NCBI servers, is the largest publicly available repository of high throughput sequencing data. The archive accepts data from all branches of life as well as metagenomic and environmental surveys. SRA stores raw sequencing data and alignment information to enhance reproducibility and facilitate new discoveries through data analysis. 	| https://www.ncbi.nlm.nih.gov/sra  	|

