<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Deseq2 Normalization and Steps | Memoirs of a bioinformatician</title>
  <meta name="description" content="This is an journaling blog for my work and reading related to bioinformatics." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Deseq2 Normalization and Steps | Memoirs of a bioinformatician" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is an journaling blog for my work and reading related to bioinformatics." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Deseq2 Normalization and Steps | Memoirs of a bioinformatician" />
  
  <meta name="twitter:description" content="This is an journaling blog for my work and reading related to bioinformatics." />
  

<meta name="author" content="Payal Banerjee" />


<meta name="date" content="2020-12-05" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="bioinformatics-databases.html"/>
<link rel="next" href="what-to-do-with-significant-differentially-expressed-genelist.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0/anchor-sections.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Memoirs of a Bioinformatician </a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> About me</a></li>
<li class="chapter" data-level="2" data-path="bioinformatics-databases.html"><a href="bioinformatics-databases.html"><i class="fa fa-check"></i><b>2</b> Bioinformatics Databases</a></li>
<li class="chapter" data-level="3" data-path="deseq2-normalization-and-steps.html"><a href="deseq2-normalization-and-steps.html"><i class="fa fa-check"></i><b>3</b> Deseq2 Normalization and Steps</a><ul>
<li class="chapter" data-level="3.1" data-path="deseq2-normalization-and-steps.html"><a href="deseq2-normalization-and-steps.html#normalization"><i class="fa fa-check"></i><b>3.1</b> Normalization</a></li>
<li class="chapter" data-level="3.2" data-path="deseq2-normalization-and-steps.html"><a href="deseq2-normalization-and-steps.html#dispersion"><i class="fa fa-check"></i><b>3.2</b> Dispersion</a></li>
<li class="chapter" data-level="3.3" data-path="deseq2-normalization-and-steps.html"><a href="deseq2-normalization-and-steps.html#generalized-linear-model"><i class="fa fa-check"></i><b>3.3</b> Generalized Linear Model</a><ul>
<li class="chapter" data-level="3.3.1" data-path="deseq2-normalization-and-steps.html"><a href="deseq2-normalization-and-steps.html#why-negative-binomeal-distribution-for-analysing-rnaseq-data"><i class="fa fa-check"></i><b>3.3.1</b> Why negative binomeal distribution for analysing RNAseq data</a></li>
<li class="chapter" data-level="3.3.2" data-path="deseq2-normalization-and-steps.html"><a href="deseq2-normalization-and-steps.html#statistical-significance-and-multiple-testing-correction"><i class="fa fa-check"></i><b>3.3.2</b> Statistical Significance and Multiple testing correction</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="deseq2-normalization-and-steps.html"><a href="deseq2-normalization-and-steps.html#references"><i class="fa fa-check"></i><b>3.4</b> References</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="what-to-do-with-significant-differentially-expressed-genelist.html"><a href="what-to-do-with-significant-differentially-expressed-genelist.html"><i class="fa fa-check"></i><b>4</b> What to do with significant differentially expressed genelist?</a><ul>
<li class="chapter" data-level="4.1" data-path="what-to-do-with-significant-differentially-expressed-genelist.html"><a href="what-to-do-with-significant-differentially-expressed-genelist.html#functional-enrichment-analysis"><i class="fa fa-check"></i><b>4.1</b> Functional Enrichment Analysis</a></li>
<li class="chapter" data-level="4.2" data-path="what-to-do-with-significant-differentially-expressed-genelist.html"><a href="what-to-do-with-significant-differentially-expressed-genelist.html#pathways"><i class="fa fa-check"></i><b>4.2</b> Pathways</a></li>
<li class="chapter" data-level="4.3" data-path="what-to-do-with-significant-differentially-expressed-genelist.html"><a href="what-to-do-with-significant-differentially-expressed-genelist.html#interaction-databases"><i class="fa fa-check"></i><b>4.3</b> Interaction Databases</a></li>
<li class="chapter" data-level="4.4" data-path="what-to-do-with-significant-differentially-expressed-genelist.html"><a href="what-to-do-with-significant-differentially-expressed-genelist.html#text-mining"><i class="fa fa-check"></i><b>4.4</b> Text Mining</a></li>
<li class="chapter" data-level="4.5" data-path="what-to-do-with-significant-differentially-expressed-genelist.html"><a href="what-to-do-with-significant-differentially-expressed-genelist.html#drug-interaction-database"><i class="fa fa-check"></i><b>4.5</b> Drug Interaction Database</a></li>
<li class="chapter" data-level="4.6" data-path="what-to-do-with-significant-differentially-expressed-genelist.html"><a href="what-to-do-with-significant-differentially-expressed-genelist.html#rna-seq-specific-databases"><i class="fa fa-check"></i><b>4.6</b> RNA seq specific Databases</a></li>
<li class="chapter" data-level="4.7" data-path="what-to-do-with-significant-differentially-expressed-genelist.html"><a href="what-to-do-with-significant-differentially-expressed-genelist.html#data-share"><i class="fa fa-check"></i><b>4.7</b> Data Share</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="genome-graphs.html"><a href="genome-graphs.html"><i class="fa fa-check"></i><b>5</b> Genome Graphs</a><ul>
<li class="chapter" data-level="5.1" data-path="genome-graphs.html"><a href="genome-graphs.html#human-reference-genome"><i class="fa fa-check"></i><b>5.1</b> Human Reference Genome</a></li>
<li class="chapter" data-level="5.2" data-path="genome-graphs.html"><a href="genome-graphs.html#pitfalls"><i class="fa fa-check"></i><b>5.2</b> Pitfalls</a><ul>
<li class="chapter" data-level="5.2.1" data-path="genome-graphs.html"><a href="genome-graphs.html#thus-there-is-reference-bias"><i class="fa fa-check"></i><b>5.2.1</b> Thus there is Reference Bias</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="genome-graphs.html"><a href="genome-graphs.html#potential-solutions"><i class="fa fa-check"></i><b>5.3</b> Potential Solutions</a></li>
<li class="chapter" data-level="5.4" data-path="genome-graphs.html"><a href="genome-graphs.html#how-to-encapsulate-all-that-information"><i class="fa fa-check"></i><b>5.4</b> How to encapsulate all that information ?</a><ul>
<li class="chapter" data-level="5.4.1" data-path="genome-graphs.html"><a href="genome-graphs.html#genome-graphs-1"><i class="fa fa-check"></i><b>5.4.1</b> <em><strong>Genome Graphs</strong></em></a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="genome-graphs.html"><a href="genome-graphs.html#existing-co-ordinate-systems"><i class="fa fa-check"></i><b>5.5</b> Existing Co-ordinate systems</a></li>
<li class="chapter" data-level="5.6" data-path="genome-graphs.html"><a href="genome-graphs.html#potential-solution---sequence-graphs"><i class="fa fa-check"></i><b>5.6</b> Potential Solution - Sequence graphs</a><ul>
<li class="chapter" data-level="5.6.1" data-path="genome-graphs.html"><a href="genome-graphs.html#things-to-consider"><i class="fa fa-check"></i><b>5.6.1</b> Things to consider:</a></li>
<li class="chapter" data-level="5.6.2" data-path="genome-graphs.html"><a href="genome-graphs.html#allelism-in-graphs"><i class="fa fa-check"></i><b>5.6.2</b> Allelism in graphs</a></li>
<li class="chapter" data-level="5.6.3" data-path="genome-graphs.html"><a href="genome-graphs.html#repeatome-as-array-sequence-graphs"><i class="fa fa-check"></i><b>5.6.3</b> Repeatome as Array Sequence Graphs</a></li>
<li class="chapter" data-level="5.6.4" data-path="genome-graphs.html"><a href="genome-graphs.html#hierarchy"><i class="fa fa-check"></i><b>5.6.4</b> Hierarchy</a></li>
<li class="chapter" data-level="5.6.5" data-path="genome-graphs.html"><a href="genome-graphs.html#pangenome-ordering"><i class="fa fa-check"></i><b>5.6.5</b> Pangenome ordering</a></li>
<li class="chapter" data-level="5.6.6" data-path="genome-graphs.html"><a href="genome-graphs.html#how-to-decipher-genome-graphs"><i class="fa fa-check"></i><b>5.6.6</b> How to decipher Genome Graphs</a></li>
<li class="chapter" data-level="5.6.7" data-path="genome-graphs.html"><a href="genome-graphs.html#extending-mapping-to-genome-graphs"><i class="fa fa-check"></i><b>5.6.7</b> Extending mapping to genome graphs</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="genome-graphs.html"><a href="genome-graphs.html#graphs-should-include"><i class="fa fa-check"></i><b>5.7</b> Graphs should include</a></li>
<li class="chapter" data-level="5.8" data-path="genome-graphs.html"><a href="genome-graphs.html#challenges-to-graph-genomes"><i class="fa fa-check"></i><b>5.8</b> Challenges to graph genomes</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="rna-fda.html"><a href="rna-fda.html"><i class="fa fa-check"></i><b>6</b> RNA-seq FDA benchmark paper</a></li>
<li class="chapter" data-level="7" data-path="single-cell-tools.html"><a href="single-cell-tools.html"><i class="fa fa-check"></i><b>7</b> Single Cell Tools</a></li>
<li class="divider"></li>
<li><a href="https://pbpayal.github.io/memoirs_of_bioinformatician/" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Memoirs of a bioinformatician</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="deseq2-normalization-and-steps" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> Deseq2 Normalization and Steps</h1>
<div id="normalization" class="section level2">
<h2><span class="header-section-number">3.1</span> Normalization</h2>
<ul>
<li>Different library sizes or Sequencing depth</li>
<li>RNA composition bias</li>
</ul>
<p>Since tools for differential expression analysis are comparing the counts between sample groups for the same gene, gene length does not need to be accounted for by the tool. However, sequencing depth and RNA composition do need to be taken into account.</p>
<p>To normalize for sequencing depth and RNA composition, DESeq2 uses the median of ratios method. On the user-end there is only one step, but on the back-end there are multiple steps involved, as described below.</p>
<p><strong>In Short:</strong></p>
<ul>
<li><p>Take geometric mean of gene's counts across all samples</p></li>
<li><p>Divide gene's counts in a sample by the geometric mean</p></li>
<li><p>Take median of these ratios -&gt; sample's normalization factor (applied to gene counts)</p></li>
</ul>
<p><strong>In Details:</strong></p>
<table>
<thead>
<tr class="header">
<th></th>
<th align="center">Sample 1</th>
<th align="right">Sample 2</th>
<th align="right">Sample 3</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Gene 1</td>
<td align="center">0</td>
<td align="right">10</td>
<td align="right">4</td>
</tr>
<tr class="even">
<td>Gene 2</td>
<td align="center">2</td>
<td align="right">6</td>
<td align="right">12</td>
</tr>
<tr class="odd">
<td>Gene 3</td>
<td align="center">33</td>
<td align="right">55</td>
<td align="right">200</td>
</tr>
</tbody>
</table>
<p><em>Step 1</em>: Log of raw base counts Log with base e</p>
<p><em>Step 2</em>: Average of the logs for each gene in each sample</p>
<p><em>Step 3</em>: Filters genes with 0 counst in more than one sample</p>
<p><em>Step 4</em>: Subtract log(raw counts) -log(average) for eacg gene This is a ratio essentially of each gene across all samples</p>
<p><em>Step 5</em>: Calculate the median for each gene</p>
<p>This helps to remove extreme gene expression like genes with high expression influencing genes with low expression. Thus focusing on genes with median expression and houskeeping genes</p>
<p><em>Step 6</em>: Convert median to normal values which is the scaling factor e^median = Normal</p>
<p><em>Step 7</em>: Divide original read counts by scaling factor</p>
</div>
<div id="dispersion" class="section level2">
<h2><span class="header-section-number">3.2</span> Dispersion</h2>
<p>When comparinng gene expression levels between groups, it is important to account for within group variabilty It is diffcult to estimate within group variabilty. Solution - pool information across genes which are expessed at similar level from replicates. Assumes that genes of similar average expression strength have similar dispersion.</p>
<ul>
<li><p><strong>Maximum Likelihood</strong> - Dispersion estimates</p></li>
<li><p>Fits a <strong>curve</strong> to capture the dependance of these estimates on the average expression strength</p></li>
<li><p>Shrinks <strong>genewise values towards the curve</strong> using an empirical Baryes approach</p></li>
</ul>
</div>
<div id="generalized-linear-model" class="section level2">
<h2><span class="header-section-number">3.3</span> Generalized Linear Model</h2>
<p>Follows negative binomeal distribution</p>
<div id="why-negative-binomeal-distribution-for-analysing-rnaseq-data" class="section level3">
<h3><span class="header-section-number">3.3.1</span> Why negative binomeal distribution for analysing RNAseq data</h3>
<p>Explained quite nicely <a href="http://bridgeslab.sph.umich.edu/posts/why-do-we-use-the-negative-binomial-distribution-for-rnaseq">here</a></p>
</div>
<div id="statistical-significance-and-multiple-testing-correction" class="section level3">
<h3><span class="header-section-number">3.3.2</span> Statistical Significance and Multiple testing correction</h3>
<p>Wald Test for significance</p>
<p>Benjamini Hochenberg</p>
</div>
</div>
<div id="references" class="section level2">
<h2><span class="header-section-number">3.4</span> References</h2>
<ol style="list-style-type: decimal">
<li><p><a href="https://youtu.be/UFB993xufUU">StatQuest: DESeq2, part 1, Library Normalization</a></p></li>
<li><p><a href="https://youtu.be/5tGCBW3_0IA">Differential expression analysis</a></p></li>
<li><p><a href="https://hbctraining.github.io/DGE_workshop/lessons/02_DGE_count_normalization.html">HCB training</a></p></li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="bioinformatics-databases.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="what-to-do-with-significant-differentially-expressed-genelist.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Payal_Blog.pdf", "Payal_Blog.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
